CHANGELOG.rst

```diff
@@ -1,7 +1,9 @@
 3.0.6.dev0
 ==========
 
-*
+* fix issue #2118 - protect against internal deprecationerrors by changing the code path of Node._getcustomclass.
```

Comment:
@RonnyPfannschmidt besides adding the link, I think it needs rewording to make more sense to users (which don't know about pytest's internals). Suggestion:

```
- pytest no longer generates ``PendingDeprecationWarning`` from its own operations, which was introduced by mistake in version ``3.0.5`` (`#2118`_).
  Thanks to `@nicoddemus`_ for the report and `@RonnyPfannschmidt`_ for the PR.
```